lib/containers/gene.js

```diff
@@ -50,38 +53,70 @@ class Gene extends React.Component {
     })
   }
 
-  renderSelectedTab = () => {
+  renderAboutSection = () => {
     switch (this.selectedTabTitle()) {
-      // case TABS.ABOUT: return <About gene={this.props.gene} />
       // case TABS.WORKS: return <Artworks gene={this.props.gene} />
       case TABS.ABOUT: return <About gene={this.props.gene} />
-      case TABS.WORKS: return <Text/>
+      case TABS.WORKS: return <Artworks gene={this.props.gene} />
     }
   }
 
-  renderTabView() {
+// Note: Foreground = the header, which gets replaced
+// Sticky = the view shown when scrolled down a lot
+
+  // When Works is active:
+  //  Foreground is title, refine selector
+  //  Sticky Header is title, follow, switch
+  //  PScrollView's children is the Artworks component
+
+  // When About is active:
+  //  Sticky Header is just gene title
+  //  PScrollView's children is the About component
```

Comment:
some notes for tomorrow, once I grok how these pieces come together these can get fleshed out, as it will be tricky to follow